web/src/app/services/navigation/navigation.service.ts

```diff
@@ -33,11 +33,10 @@ import {map} from 'rxjs/operators';
   providedIn: 'root',
 })
 export class NavigationService {
-  private static readonly JOB_ID_FRAGMENT_PARAM = 'l';
-  private static readonly LOI_ID_FRAGMENT_PARAM = 'f';
-  private static readonly LOI_JOB_ID_FRAGMENT_PARAM = 'fl';
-  private static readonly SUBMISSION_ID_FRAGMENT_PARAM = 'o';
-  private static readonly TASK_ID_FRAGMENT_PARAM = 't';
+  private static readonly JOB_ID_FRAGMENT_PARAM = 'job';
+  private static readonly LOI_ID_FRAGMENT_PARAM = 'site';
+  private static readonly SUBMISSION_ID_FRAGMENT_PARAM = 'submission';
+  private static readonly TASK_ID_FRAGMENT_PARAM = 'task';
```

Comment:
Hi @amysorto ! This will rename the fragments, but we also need the router to use the URI rather than URL fragment. The exact patterns are described in the issue descriptions. Please lmk if you have any questions!